controller/metrics/metrics.go

```diff
@@ -92,6 +94,17 @@ func NewMetricsServer(addr string, appLister applister.ApplicationLister, health
 	)
 	registry.MustRegister(k8sRequestCounter)
 
+	kubectlExecCounter := prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "argocd_kubectl_exec_total",
+		Help: "Number of kubectl executions",
+	}, []string{"command"})
+	registry.MustRegister(kubectlExecCounter)
+	kubectlExecPendingGauge := prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "argocd_kubectl_exec_pending",
```

Comment:
As per prometheus naming guidelines, this should also be suffixed with `_total` for counters. Maybe `argocd_kubectl_exec_pending_total` or `argocd_kubectl_pending_exec_total`

Reply:
https://prometheus.io/docs/practices/naming/
https://www.robustperception.io/on-the-naming-of-things

Reply:
Also can you rename: `argocd_cluster_events_count` to `argocd_cluster_events_total` ?

Reply:
done